internal/controller/template.go

```diff
@@ -35,6 +35,46 @@ const krakendConfigTemplate = `{
         "router": {
             "disable_access_log": false,
             "hide_version_header": true
+        },
+        "@comment": "OpenTelemetry configuration for distributed tracing. trace_sample_rate defaults to 1.0 (100%).",
+        "telemetry/opentelemetry": {
+            "service_name": "{{.DeploymentName}}",
+            "service_version": "{{.ServiceVersion}}",
+            "exporters": {
+                "otlp": [
+                    {
+                        "name": "otel-collector",
+                        "host": "{{.OtelCollectorHost}}",
```

Comment:
How about this feature? https://www.krakend.io/docs/configuration/environment-vars/#environment-variables-with-flexible-configuration